appveyor.yml

```diff
@@ -1,5 +1,3 @@
-# http://www.appveyor.com/docs/appveyor-yml
```

Comment:
these changes were automatically added when running tests. Not familiar with this file. Is this okay?